src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -66,6 +66,9 @@ abstract contract DelegationManagerStorage is IDelegationManager {
      */
     mapping(address => mapping(bytes32 => bool)) public delegationApproverSaltIsSpent;
 
+    // @notice Mapping: staker => UndelegationLimboStatus struct. Mapping is internal so we can have a getter that returns a memory struct.
+    mapping(address => IDelegationManager.UndelegationLimboStatus) internal _stakerUndelegationLimboStatus;
```

Comment:
undelegation limbo might top pubkey compendium in terms of obnoxious naming

Reply:
yeah can we rename to Queue or something similar

Reply:
so this is a state where a staker is essentially "waiting to finalize / fully enact" their undelegation from an operator. I think I suggested "purgatory" previously but don't like the associations.
'UndelegationQueue' could potentially work, although we've also had confusion around the word 'queue' with people perceiving these things to have some kind of ordering to them.

Reply:
i meant this as a complement @stevennevins